.github/workflows/pre-commit-hooks.yml

```
@@ -12,6 +12,6 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.x'
+          python-version: '3.8'
```

```suggestion
          python-version: '3.10'
```
This is the version I have installed locally, and it works fine.